### PR TITLE
Add match ug

### PR DESCRIPTION
**Pull request by Chrainx**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -106,19 +77,48 @@ Parameter:
...
+* `New property added:Name; Address:ADDRESS; Phone:PHONE; Price:price; Tags:[TAGS]…​`
+
+When command fails:
+* `This property already exist` if the property have the same Name and Address
+* `Missing Name parameter for add properties command` for missing Name parameter
+* `Missing Address parameter for add properties command` for missing Address parameter
+* `Missing number parameter for add properties command` for missing Name parameter
+* `Missing price parameter for add properties command` for missing price parameter
+* `Invalid Command` for mispelling of command
+
```
Comment (jianrong7) [inline docs/UserGuide.md:121]: Why did you change `addprop` and put it after `addcust`?

**Review by jianrong7 — APPROVED** [review]
okay good, but be careful there are some conflicts with the `Command Summary` table. I think it is better to follow the one currently on `master` instead of your changes because it is more updated.

Nevertheless, thanks for the update! 👍

File: docs/UserGuide.md [hunk 2]
```diff
@@ -106,19 +77,48 @@ Parameter:
...
 * `Invalid Command` for misspelling of command
 
+### Adding a property: `addprop`
+
+Adds a property to the application.
+
+Format: `addprop n/NAME a/ADDRESS [c/CHARACTERISTIC] ph/number pr/budget`
+
+Parameter:
+* `n/NAME`				         : The propName of the property (String)
```
Comment (saraozn) [inline docs/UserGuide.md:100]: I think `propName` can be changed to `name` instead, users probably wouldn't understand `propName`

File: docs/UserGuide.md [hunk 3]
```diff
@@ -106,19 +77,48 @@ Parameter:
...
+* `a/ADDRESS`		             : The propAddress of the property (String)
```
Comment (saraozn) [inline docs/UserGuide.md:101]: likewise for this!

**Review by saraozn — APPROVED** [review]
looks good, just a few suggestions
